### PR TITLE
[ja] Localize Policy as Code (PaC) into Japanese

### PR DESCRIPTION
**Pull request by t-katsumura**
### Describe your changes

Localize https://glossary.cncf.io/policy-as-code/ into Japanese

### Related issue number or link (ex: `resolves #issue-number`)

Fix https://github.com/cncf/glossary/issues/2825

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ja/policy-as-code.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Policy as Code (PaC)
+status: Completed
+category: concept
```
Comment (Okabe-Junya) [inline content/ja/policy-as-code.md:4]:
```suggestion
category: コンセプト
```

ここも翻訳をお願いします

**Comment by Okabe-Junya** [comment]
The target page on the deploy preview:

- https://deploy-preview-2992--cncfglossary.netlify.app/ja/policy-as-code/

**Review by Okabe-Junya — CHANGES_REQUESTED** [review]
コントリビューションありがとうございます〜！

一点だけ軽微な修正をお願いしたいです

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Comment by t-katsumura** [comment]
@Okabe-Junya 
ありがとうございます！
修正取込みました

**Pull request merged**